index.md

```diff
@@ -11,7 +11,7 @@ experience in particular with microservices and
 [Self-contained Systems](http://scs-architecture.org) and the
 challenges faced in those projects.
 
-Far too often microservices are adopted but project fail to
+Far too often microservices are adopted but the projects then fail to
```

Comment:
I’m still not too happy with this. Maybe this?

> …are adopted in projects that then fail to succeed.

It makes the connection between the projects and microservices clearer, but it eliminates the “but” which I liked. :-/

Reply:
I like both. Not sure which one is better.

Reply:
Let’s stick with “but” then, emphasizing that the projects fail in spite of using microservices.